src/widget/flex.rs

```diff
@@ -165,21 +169,28 @@ impl<T: Data> Widget<T> for Flex<T> {
             }
         }
 
-        let total_major = self.direction.major(bc.max);
-        let remaining = total_major - total_non_flex;
+        let total_major = self.direction.major(bc.max());
+        let remaining = (total_major - total_non_flex).max(0.0);
```

Comment:
It would create constraints of negative size. Often these would go through a `Rect` constructor that would take `abs`, so you'd get this sorta "bounce" effect.